### PR TITLE
Branch: jira-2386-add-search-test-cases/

### PR DESCRIPTION
**Pull request by akashdktyagi**

### CODE REVIEW

**Review by sarangholey — COMMENTED** [review]
Ok letz go for a change

File: src/test/java/com/visionit/TestClass2.java [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 public class TestClass2 {
 
+    //Added by akash Done as a part of this ticket: jira-2386-add-search-test-cases/
+    public void test_00(){
```
Comment (sarangholey) [inline src/test/java/com/visionit/TestClass2.java]: Change needed here on the line

Comment (akashdktyagi) [reply src/test/java/com/visionit/TestClass2.java]: ok i will make the cahnges.

Comment (akashdktyagi) [reply src/test/java/com/visionit/TestClass2.java]: changes are made, resolving the conversation.

Comment (sarangholey) [reply src/test/java/com/visionit/TestClass2.java]: Ok Akash Sir, go for it

**Pull request merged**